### PR TITLE
This is the best commit, the greatest commit.

### PR DESCRIPTION
**Pull request by nadav7679**

### CODE REVIEW

File: yaml/nadav/yamel_exercise1_nadav.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# string
+name: Nadav
+# number
+age: 21
```
Comment (etayruben) [inline yaml/nadav/yamel_exercise1_nadav.yaml:4]: you are very old

**Comment by noamu111** [comment]
I am the stupid.

File: yaml/nadav/yamel_exercise1_nadav.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# array option 2
+friends:
+   - name: Aviv
+     age: 21
+     isPerson: true
+     isOlder: true
+
+    - name: Noy
+      age: 21
+      isPerson: true
```
Comment (etayruben) [inline yaml/nadav/yamel_exercise1_nadav.yaml:17]: not true she is a psycho

**Review by etayruben — CHANGES_REQUESTED** [review]

File: yaml/nadav/yamel_exercise1_nadav.yaml (hunk 2 above)
Comment (nadav7679) [reply yaml/nadav/yamel_exercise1_nadav.yaml:17]: u stupid 
it's supposed to be "Itay", with an 'I'.

**Comment by baarzelim** [comment]
> I am the stupid.

where were you when clab pinguin death? i was at house eating doritos, phone ring. clab pinguin is kill, NO

File: yaml/nadav/yamel_exercise1_nadav.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+         isOlder: true
+         dateOfBirth: 4.5.1967
+
+       Alon:
+         kids: 3
+         age: 58
+         isOlder: true
+         dateOfBirth: 13.8.1962
+
+summery: >
```
Comment (benedery) [inline yaml/nadav/yamel_exercise1_nadav.yaml]: fix typo

**Review by benedery — CHANGES_REQUESTED** [review]

**Pull request merged**